Sources/SmileID/Classes/Metadata/DeviceOrientationMetadataProvider.swift

```diff
@@ -7,8 +7,12 @@ class DeviceOrientationMetadataProvider: MetadataProvider {
 
     private let motionManager = CMMotionManager()
 
+    private struct OrientationEvent {
+        let value: String
+        let date: Date
```

Comment:
Can we use the standardised date function for this? https://github.com/smileidentity/ios/blob/2c29b07496ae29b764403921bb1f633e1727002a/Sources/SmileID/Classes/Helpers/DateUtils.swift#L6. Ideally we set this as default value, which reduces the amount of code we need to write.
```suggestion
        let date: Date = Date().toISO8601WithMilliseconds()
```